.github/workflows/ci.yml

```diff
@@ -36,12 +36,12 @@ jobs:
         with:
           components: rustfmt, clippy
 
-      - name: Install cargo-deny
+      - name: Install cargo-deny and just
         if: steps.rust-cache.outputs.cache-hit != 'true'
-        run: rustup run --install 1.74 cargo install --force --version 0.14.3 cargo-deny --locked
-
+        run: rustup run --install 1.74 cargo install --force --version 0.14.3 cargo-deny --locked && cargo install --version 1.13.0 just --locked
```

Comment:
You need to use `rustup run --install 1.85 cargo install just ...` to install just. Otherwise it is going to fail because of the pinned Rust version in rust-toolchain file